packages/button/src/button.tsx

```diff
@@ -126,7 +131,7 @@ export const Button = forwardRef<ButtonProps, "button">((props, ref) => {
       {leftIcon && !isLoading && (
         <ButtonIcon marginEnd={iconSpacing}>{leftIcon}</ButtonIcon>
       )}
-      {isLoading && (
+      {isLoading && loadingSpinnerPosition === "start" && (
         <ButtonSpinner
           __css={{ fontSize: "1em", lineHeight: "normal" }}
```

Comment:
- Consider moving this style `{ fontSize: "1em", lineHeight: "normal" }` to `ButtonSpinner` component itself